src/interface_r/vignettes/getting_started.Rmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "H2O4GPU: Machine Learning with GPUs in R"
+author: "Navdeep Gill, Erin LeDell, Yuan Tang"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+
+
+## Installation
+
+First, please follow the instruction [here](https://github.com/h2oai/h2o4gpu#installation) to build the H2O4GPU Python package.
+
+Then install `devtools` R package via `install.packages("devtools")` if you haven't done it yet.
+
+Finally, we can install `h2o4gpu` R package via the following:
+
+```
+library(devtools)
+devtools::install_github("h2oai/h2o4gpu", subdir = "src/interface_r")
+```
+
+## Train & Evaluate a Model
+
+Here's a quick demo of how to train and evaluate a GPU-based Random Forest model:
+```{r, echo=FALSE, results='foo'}
+library(h2o4gpu)
+
+# Setup dataset
+x <- iris[1:4]
+y <- as.integer(iris$Species) - 1
+# TO DO: use a different dataset and split into train/test
+
+# Initialize and train the classifier
+model <- h2o4gpu.random_forest_classifier() %>% fit(x, y)
+
+# Make predictions
+predictions <- model %>% predict(x)
+
+# Score the model (uses Python scikit-learn score)
+# Not sure yet if we should keep score()
```

Comment:
h2o4gpu Python package depends on sklearn right? If sklearn is installed along the way I would say we keep this. @navdeep-G What do you think?